scripts/vitest/setupFiles/lodestarPreset.ts

```diff
@@ -1,11 +1,12 @@
 import {PresetName, setActivePreset} from "@lodestar/params/setPreset";
+
 // Set minimal
 if (process.env.LODESTAR_PRESET === undefined) {
   process.env.LODESTAR_PRESET = "minimal";
 }
 
 // Override FIELD_ELEMENTS_PER_BLOB if its a dev run, mostly to distinguish from
 // spec runs
-if (process.env.LODESTAR_PRESET === "minimal" && process.env.DEV_RUN) {
+if (process.env.LODESTAR_PRESET === "minimal") {
```

Comment:
Curious about this.  Never noticed this `DEV_RUN` before.  What was it doing and are you sure we dont need it anymore?  Like its not for CI or for anything DX?